plutus-pab/src/Cardano/Node/Params.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE NamedFieldPuns #-}
+module Cardano.Node.Params where
```

Comment:
Nice, this is much better!

Did you need this change, or is it just something you noticed?

Reply:
I noticed several `readPP` functions and decided to make a proper one.